src/main/java/com/frcteam3636/frc2024/subsystems/drivetrain/Drivetrain.kt

```diff
@@ -186,6 +206,9 @@ class DrivetrainIOSim : DrivetrainIO() {
 internal val WHEEL_BASE: Double = Units.inchesToMeters(13.0)
 internal val TRACK_WIDTH: Double = Units.inchesToMeters(14.0)
 
+internal val ROTATION_PID_CONTROLLER = PIDController(PIDGains(0.3, 0.0, 0.0))
```

Comment:
We don't want this variable to keep state, so the module-level variable should just be the gains and the controller itself should be moved into a property of the subsystem.